background.js

```diff
@@ -1,31 +1,66 @@
 "use strict";
 
-/* global chrome, URL */
+/* global chrome */
 
 chrome.runtime.onInstalled.addListener(function () {
-  chrome.storage.local.get(["blocked", "enabled"], function (local) {
-    if (!Array.isArray(local.blocked)) {
-      chrome.storage.local.set({ blocked: [] });
-    }
-
+  chrome.storage.local.get(["enabled", "blocked"], function (local) {
     if (typeof local.enabled !== "boolean") {
       chrome.storage.local.set({ enabled: false });
     }
+
+    if (!Array.isArray(local.blocked)) {
+      chrome.storage.local.set({ blocked: [] });
+    }
   });
 });
 
+// "https://www.youtube.com/" => "youtube.com"
+// "https://www.youtube.com/feed/explore" => "youtube.com/feed/explore"
+// "https://music.youtube.com/" => "music.youtube.com"
+// "https://music.youtube.com/explore" => "music.youtube.com/explore"
+const getDomainUrl = (url) => {
+  let domainUrl = url.replace(/^http(s?):\/\//, "").replace(/^www\./, "");
+  domainUrl = domainUrl.endsWith("/") ? domainUrl.slice(0, -1) : domainUrl;
+  return domainUrl;
+};
```

Comment:
`www.` is not important (and is stripped), other subdomains are important, like `music.`.